src/BluetoothContext.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from "react";
+import { BluetoothConnections, BluetoothNotificationsStatus } from "./types/Bluetooth";
+
+export const BluetoothContext = React.createContext<BluetoothConnections>({
```

Comment:
See https://kentcdodds.com/blog/how-to-use-react-context-effectively for why a default value should **not** be defined for the context

In my own words, this will mask incorrect usage of the provider and useBluetooth hook.  If we provide no default (undefined) then the useBluetooth hook will throw a nice error if called from outside the BluetoothProvider component.  

This simply becomes
```
export const BluetoothContext = React.createContext<BluetoothConnections | undefined>(undefined)
```